packages/react-ui-components/src/Button/button.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import mergeClassNames from 'classnames';
-import {makeFocusNode} from './../_lib/focusNode';
+import { makeFocusNode } from './../_lib/focusNode';
```

Comment:
No need for extra spaces here, breaks our code style

Reply:
removed the spaces (was something phpStorm did automatically and I didn't even notice it...)
also removed the changes to yarn.lock ;)